tests/test_property.py

```diff
@@ -17,14 +17,14 @@
 def test_instantiate_property() -> None:
     """It returns a TypeErro exception."""
```

Comment:
Denne doc-stringen er feil. Poenger her er vel å test at instansering _ikkje_ kaster exepction. Så korrekt doc-string er: 
```
"""It does not raise an exception."""
```
Dette gjelder for alle testene av denne typen.

Reply:
Hei @stigbd , takk for tips, skal fikse dette! :+1:

Reply:
Da har jeg fikset dette. :-)